src/nsidc/metgen/metgen.py

```diff
@@ -346,8 +346,7 @@ def process(configuration: config.Config) -> None:
             browse_filenames=browse_files,
             premet_filename=premet_file,
             spatial_filename=spatial_file,
-            reference_data_filename=reference_data_file,
-            data_reader=data_reader(configuration.auth_id, reference_data_file),
+            data_reader=data_reader(data_files),
```

Comment:
Your change is reverting a patch made for issue-169. The original lines 349 and 350 are correct.